.circleci/config.yml

```diff
@@ -112,8 +112,8 @@ jobs:
             - $HOME/go/src/
 
       # The weird way you have to set PATH in Circle 2.0
-      - run: echo 'export PATH=$HOME/.local/bin:$HOME/go/bin:$HOME/terraform:$HOME/packer:$PATH'
-          >> $BASH_ENV
+      - run: |
```

Comment:
What difference does it make to use the `|`?

Reply:
`|` ensures the string is treated as "raw" multiline, and prevents scripts from reflowing the text to keep column width.